src/js/routes/Ballot/Ballot.jsx

```diff
@@ -434,15 +469,14 @@ export default class Ballot extends Component {
         </div> : null
       */}
       {emptyBallot}
-      <div className="BallotList">
-        { in_ready_to_vote_mode ?
-          ballot.map( (item) => <BallotItemReadyToVote key={item.we_vote_id} {...item} />) :
-          ballot.map( (item) => <BallotItemCompressed _toggleCandidateModal={this._toggleCandidateModal}
-                                                      _toggleMeasureModal={this._toggleMeasureModal}
-                                                      key={item.we_vote_id}
-                                                      {...item} />)
-        }
-      </div>
-    </div>;
+
```

Comment:
I think it would be good to leave BallotList in place unless you have a reason to remove it.